.github/workflows/deploy-docker.yml

```diff
@@ -36,18 +35,33 @@ jobs:
             -
                 name: Get branch name
                 id: branch-name
-
                 uses: tj-actions/branch-names@v6
 
 
             -
                 name: Set tag for non-default branch
-                if: steps.branch-name.outputs.is_default == 'false' && steps.branch-name.outputs.default_branch != ''
+                if: ${{ steps.branch-name.outputs.is_default == 'false' && steps.branch-name.outputs.default_branch != '' && github.actor != 'dependabot[bot]' }}
                 run: |
+                    echo "Run me: (${{ steps.branch-name.outputs.is_default == 'false' && steps.branch-name.outputs.default_branch != '' && github.actor != 'dependabot[bot]' }})"
+                    echo "Set tag for PR: ${{ github.event_name == 'pull_request' || github.actor == 'dependabot[bot]' && steps.branch-name.outputs.is_default == 'false' && steps.branch-name.outputs.default_branch != '' }} "
+                    echo "Actor = ${{ github.actor }}"
+                    echo "event name is ${{ github.event_name }}"
```

Comment:
This is more for debugging than anything. But as this runs as an action it may be helpful to see these values sometimes. Thats why they arent removed.